README.md

```diff
@@ -21,7 +21,7 @@ yarn add -D eslint-config-mig-react-native
 Install the package peer dependencies as development depency (if you don't have them allready).
 
 ```bash
-yarn add -D eslint eslint-config-prettier eslint-plugin-detox eslint-plugin-import eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-react-hooks prettier prettier-eslint-cli @react-native-community/eslint-config typescript @typescript-eslint/parser @typescript-eslint/eslint-plugin
+yarn add -D eslint eslint-config-prettier eslint-plugin-detox eslint-plugin-import eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-react-hooks prettier prettier-eslint-cli @react-native-community/eslint-config typescript @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-plugin-simple-import-sort
 ```
```

Comment:
I propose to add them as dependencies, so that they are installed with the package instead of a manual action. Not blocking imo just a nice to have 👍

Reply:
I agree, but I felt it was a bit too impactful for this PR and should be discussed as a separate proposal.